package.json

```diff
@@ -24,11 +24,13 @@
     "cypress:open": "cypress open",
     "cypress:run": "cypress run",
     "test": "jest",
+    "coverage": "jest --coverage && browser-sync start --server \"coverage/lcov-report\"",
```

Comment:
The script combines the jest coverage command with the browser-sync server command. Splitting these into separate scripts could offer more flexibility. For instance, developers might want to generate coverage without serving it on a browser or serve an existing report without regenerating it.
So, what about refactoring the scripts like this:

```
"scripts": {
  "test:coverage": "jest --coverage",
  "coverage:report": "browser-sync start --server \"coverage/lcov-report\"",
  "coverage": "npm run test:coverage && npm run coverage:report"
}

```

Reply:
Thank you, I completely agree with your idea of refactoring so will change package json in the next PR!